tck/base/pom.xml

```diff
@@ -18,7 +18,7 @@
     <parent>
         <artifactId>microprofile-opentracing-tck-parent</artifactId>
         <groupId>org.eclipse.microprofile.opentracing</groupId>
-        <version>2.1-SNAPSHOT</version>
+        <version>3.0-SNAPSHOT</version>
```

Comment:
The <build> element from the tck pom.xml should be removed as well, I think.

Reply:
Just push an update to the commit for removing the `<build>` element in pom.xml

Reply:
What I meant is to remove `<build>` elment from the tck\pom.xml.